client/src/components/AccountCard.tsx

```diff
@@ -74,10 +77,15 @@ export default function AccountCard(props: Props) {
           confirmedAmount
         );
         props.updateAppFund(appFunds[0]);
+        const { data: newAccount } = await incrementTransfersByAccount(
```

Comment:
Hm, it might make more sense for this increment logic to just be done in the backend whenever you call `updateAppFundsBalance` since it seems like that API call would always want to increment the `number_of_transfers` field anyway. This way the client doesn't have to maintain updating the source of truth on that number.

Reply:
good call. will do. thanks.